improver_tests/psychrometric_calculations/hail_size/test_hail_size.py

```diff
@@ -312,3 +280,80 @@ def test_model_id_attr(
         relative_humidity_on_pressure,
     )
     metadata_check(result)
+    cube_size_check(result)
+
+
+def test_re_ordered_cubes(
+    temperature_on_pressure_levels,
+    ccl_pressure,
+    relative_humidity_on_pressure,
+    ccl_temperature,
+):
+
+    """Tests Plugin is input cubes have coordinates in a different order.
+    Checks that the outputted cube has coordinates in the same order as the inputs"""
```

Comment:
The first line of this is not quite a sentence.

Reply:
done